packages/shared/tsconfig.build.json

```diff
@@ -4,5 +4,5 @@
     "outDir": "./dist",
     "types": ["node"]
   },
-  "include": ["./src"]
+  "include": ["./src", "./tests"]
```

Comment:
Why have you included `./tests` in the production code?

Reply:
If this is included this the dist folder is incorrect:

![image](https://github.com/stemmlerjs/ddd-forum-bp-first/assets/85180621/4dc374f9-d9bf-4916-bdfe-2c325644b7fa)